packages/fast-development-site-react/src/components/site/dev-tools-code-preview.tsx

```diff
@@ -0,0 +1,298 @@
+        if (property !== "children") {
+            if (isObject(data[property])) {
+                return this.getReactAttributeObject(location, property, data, tab);
+            } else if (typeof data[property] === "string") {
+                return `\n${tab}${property}="${data[property]}"`;
+            } else if (typeof data[property] !== "undefined") {
+                return `\n${tab}${property}={${data[property]}}`;
+            }
+        }
+
+        return "";
+    }
+
+    /**
+     * Gets the attributes as a string
+     */
+    private getReactAttributeObject(location: string, property: string, data: any, tab: string): string {
+        const propertyName: string = location === ""
+            ? property
+            : location + property.charAt(0).toUpperCase() + property.slice(1);
+        const childrenLocations: string[] = this.findChildren(data[property], "", []);
+        const propertyChildren: IReactChildren[] = this.getReactChildrenFromProperties(data, property, propertyName, childrenLocations);
+        const attributes: string = `\n${tab}${property}={${propertyName}}`;
+
+        this.variables += `var ${propertyName} = `;
+        this.variables += `${this.replaceReactPropertyChildren(data[property], propertyChildren)};\n\n`;
+        this.setReactChildrenVariables(childrenLocations, propertyChildren, propertyName);
+
+        return attributes;
+    }
+
+    private getReactChildrenFromProperties(
+        data: any,
+        property: string,
+        propertyName: string,
+        childrenLocations: string[]
+    ): IReactChildren[] {
+        const propertyChildren: IReactChildren[] = [];
+
+        if (childrenLocations.length > 0) {
+            for (let i: number = childrenLocations.length - 1; i >= 0; i--) {
+                propertyChildren.push({
+                    id: uniqueId(),
+                    propertyName,
+                    location: childrenLocations[i],
+                    data: get(data, `${property}.${childrenLocations[i]}.children`)
+                });
+            }
+        }
+
+        return propertyChildren;
+    }
+
+    /**
+     * Sets the private variables to children objects
+     */
+    private setReactChildrenVariables(childrenLocations: string[], propertyChildren: IReactChildren[], propertyName: string): void {
+        if (childrenLocations.length > 0) {
+            for (let i: number = childrenLocations.length - 1; i >= 0; i--) {
+                let stringDataWithReactComponentNames: string = JSON.stringify(propertyChildren[i].data.props, null, 2);
+
+                stringDataWithReactComponentNames = this.replaceReactChildrenVariableIdsWithComponent(
+                    childrenLocations,
+                    propertyChildren,
+                    stringDataWithReactComponentNames
+                );
+
+                this.variables += `var ${propertyName}${propertyChildren[i].location.replace(/children|props|\./g, "")} = `;
```

Comment:
`const` is pretty universally supported these days - we can probably use it here.